inc/class-rewrite.php

```diff
@@ -159,4 +171,34 @@ public function category_rewrite_rules() {
 
 		return $category_rewrite;
 	}
+	/**
+	 * Convert encoded URI string to uppercase.
+	 * 
+	 * @return string
+	 */
+	public function encodedToUpper( &$encoded ) {
```

Comment:
You are returning this value, so this reference (`&`) isn't needed.

Reply:
I've applied all changes you suggested. Concerning the reference here, I think it's needed for array_walk() to work correctly. I'll changed this method to return bool instead of string.